pkg/databind/internal/secrets/command.go

```diff
@@ -18,7 +18,7 @@ import (
 type Command struct {
 	Path           string   `yaml:"path"`
 	Args           []string `yaml:"args,omitempty"`
-	PassthroughEnv []string `yaml:"passthrough_environment,omitempty"`
+	PassthroughEnv []string `yaml:"passthrough_environment,omitempty"` //nolint:tagliatelle
```

Comment:
I suggest adding this ignore rule at the project level

Reply:
Makes total sense